Sat 30 Nov 2024 19:19:21 IST

SLURM_JOBID:		 1596142
SLURM_JOB_NODELIST:	 ise-4090-02 


248 753 46
Traceback (most recent call last):
  File "/cs_storage/alonkay/Thesis/Dataset.py", line 659, in <module>
    remove_bad_monkaa()
  File "/cs_storage/alonkay/Thesis/Dataset.py", line 633, in remove_bad_monkaa
    for img0, img1, F, pts1, pts2, _, img_name, seq_path in train_loader:
  File "/home/alonkay/.conda/envs/alon_env/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 701, in __next__
    data = self._next_data()
  File "/home/alonkay/.conda/envs/alon_env/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 757, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/home/alonkay/.conda/envs/alon_env/lib/python3.9/site-packages/torch/utils/data/_utils/fetch.py", line 55, in fetch
    return self.collate_fn(data)
  File "/cs_storage/alonkay/Thesis/Dataset.py", line 147, in custom_collate_fn
    imgs1, imgs2, Fs, all_pts1, all_pts2, seq_names = zip(*batch)
ValueError: too many values to unpack (expected 6)
